plugin/api.go

```diff
@@ -152,6 +152,27 @@ type API interface {
 	// Minimum server version: 5.6
 	GetUsersInTeam(teamId string, page int, perPage int) ([]*model.User, *model.AppError)
 
+	// GetPreferencesForUser gets a user's preferences.
+	//
+	// @tag User
+	// @tag Preference
+	// Minimum server version: 5.24
```

Comment:
@lieut-data This feature has missed 5.24.  Should this change to 5.25?

Reply:
```suggestion
 	// Minimum server version: 5.26
```